src/config.rs

```diff
@@ -125,7 +153,7 @@ impl PdbConfigSection {
                 .unwrap_or(&json::Json::Null) {
                     &json::Json::Array(ref urls) => Some(urls.into_iter().map(|url| url.as_string().unwrap().to_string()).collect::<Vec<String>>()),
                     &json::Json::String(ref urls) => Some(split_server_urls(urls.clone())),
-                    &json::Json::Null => Some(default_server_urls()),
+                    &json::Json::Null => None,
```

Comment:
We already do this defaulting elsewhere and it just gets in the way here when trying to do fallback correctly (merging configs).